porcupine/tabs.py

```diff
@@ -631,15 +641,25 @@ def __init__(
         )
 
         # I don't know why this needs a type annotation for self.panedwindow
-        self.panedwindow: utils.PanedWindow = utils.PanedWindow(self, orient="horizontal")
+        self.panedwindow: utils.PanedWindow = utils.PanedWindow(
+            self, orient="horizontal", borderwidth=0
+        )
         self.panedwindow.pack(side="left", fill="both", expand=True)
 
         # we need to set width and height to 1 to make sure it's never too
         # large for seeing other widgets
         #
         # I don't know why this needs a type annotation for self.textwidget
         self.textwidget: textutils.MainText = textutils.MainText(
-            self, width=1, height=1, wrap="none", undo=True, padx=3
+            self,
+            width=1,
+            height=1,
+            wrap="none",
+            undo=True,
+            padx=3,
+            borderwidth=0,
+            relief="solid",
+            highlightthickness=0,
```

Comment:
Some of these changes should be reflected in the minimap plugin. Currently it does not line up nicely with the main text widget:

![Screenshot at 2021-08-24 15-30-56](https://user-images.githubusercontent.com/18505570/130616785-9cb190a5-c3b7-4eca-9857-54243b816c02.png)

What is the reason for changing all this in the first place? What was wrong with the default values?

Reply:
lol, it wasn't intended, I just played around with the border, because I don't like the `highlightthickness`, but I didn't want to commit it

Reply:
You might like `git diff --cached` (or `--staged`) before committing so you actually see what you are about to commit. To include only a part of a file for committing, you can use `git add -p file.py`, and it will ask about each change whether you want it to be in the commit or not.

Reply:
The funny thing is that I did `git diff --staged` but didn't notice